spec/feature_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'takeaway'
+require 'menu'
+
+ describe "Features" do
+   let(:takeaway) { Takeaway.new }
+   let(:x) { {"pizza"=>3, "curry"=>4, "chopsuey"=>2} }
```

Comment:
`x`?!?!

do not ever use `x` as a name - names should be descriptive! Why not call it `items` or `menu` or anything at all, just not `x`.